src/Pages/SongPage/FloatingButtons/FloatingButtons.tsx

```diff
@@ -55,34 +52,34 @@ export default function FloatingButtons({
                     >
                         {colorMode === "light" ? "Dark mode" : "Light mode"}
                     </Menu.Item>
-                    <Menu.Item
-                        leftSection={
-                            <IconDownload
-                                style={{ width: rem(14), height: rem(14) }}
-                            />
-                        }
-                        onClick={downloadJson}
-                    >
-                        Download JSON
-                    </Menu.Item>
-                    {/* <Menu.Item
-                        leftSection={
-                            <IconGitCompare
-                                style={{ width: rem(14), height: rem(14) }}
+                    <Menu.Item>
+                        <div style={{ width: 200, height: 35 }}>
+                            <Slider
+                                value={textSize}
+                                onChange={(size) => setTextSize(size)}
+                                min={0.8}
+                                max={2.0}
+                                step={0.1}
+                                marks={[
+                                    {
+                                        value: 0.8,
+                                        label: (
+                                            <b style={{ fontSize: "0.7em" }}>
```

Comment:
Should this be the same size as line 75 ?

Reply:
Well, those two are the labels for the range slider which changes the text size (The beginning and end where there are the letter A's of two different sizes)

![Screenshot_20240202-143703_Opera](https://github.com/Faith-UnFeigned/cis-web/assets/78272760/c2eba0d4-827a-48d7-9e42-e4283085d32f)